### DISCUSSION TITLE
I'm getting "Import Error" when installing poetry

### DISCUSSION THREAD

**Discussion by chrismyan1** [OpenBB-finance/OpenBB#832] [Q&A] [answered]
When I try to execute "poetry install -E prediction" I get the error below. I think I got the same error when running "install poetry" but I just moved on

ImportError

  DLL load failed while importing shell: The specified procedure could not be found.

  at C:\ProgramData\Anaconda3\envs\gst\lib\site-packages\appdirs.py:480 in _get_win_folder_with_pywin32
      476│     return dir
      477│
      478│
      479│ def _get_win_folder_with_pywin32(csidl_name):
    → 480│     from win32com.shell import shellcon, shell
      481│     dir = shell.SHGetFolderPath(0, getattr(shellcon, csidl_name), 0, 0)
      482│     # Try to make this a unicode path because SHGetFolderPath does
      483│     # not return unicode strings when there is unicode data in the
      484│     # path._

I check that file path from the error and I found the appdirs.py file so I'm not sure why it's throwing that error. I'm fairly new to python so please excuse my ignorance

Note for some reason it installed the GamestonkTerminal in the System32 folder. Could this be causing the issue?

**Comment by aia** [comment, marked as answer]
Unfortunately, poetry doesn't handle Windows for me very well either.

On Windows, I use Anaconda and pull dependencies via pip.

I run poetry in WSL.
